Gemfile

```diff
@@ -1,3 +1,15 @@
 source "https://rubygems.org"
 
 gemspec
+
+group :development do
+  gem 'byebug'
+end
+
+group :test do
+  %w[ rspec rspec-core rspec-expectations rspec-mocks rspec-support ].each do |lib|
+    gem lib, github: "rspec/#{lib}"
+  end
```

Comment:
I just matched with Vanguard. It is easier for me, because I am learning by example, and V. is before my eyes daily.